src/js/containers/bulkDownload/awards/AwardDataContainer.jsx

```diff
@@ -183,6 +191,22 @@ export class AwardDataContainer extends React.Component {
         this.props.clearDownloadFilters('awards');
     }
 
+    bulkPrimeAwardTypeChange(selection) {
```

Comment:
You could persuade me on this; but it seems to me we could condense these 4 actions into one action that takes multiple parameters? This is a lot but you could persuade me that it is necessary or more elegant to do it this way.

Reply:
Condensed down to two actions and reducers (one for bulk and one for a single toggle).